src/Vegetation/canopy_parameterizations.jl

```diff
@@ -445,3 +450,86 @@ function medlyn_conductance(
     gs = g0 + Drel * medlyn_term * (An / ca)
     return gs
 end
+
```

Comment:
I wonder if we could move the bulk albedo and emissivity to the next PR? we dont actually use them in this PR, yet, but we will need them as soon as we combine with the soil model.